examples/liquidation_bot/config.yml

```diff
@@ -0,0 +1,167 @@
+version: '0.1'
+guardians:
+  ethereum:
+    networkType: ethereum
+    network: kovan
+    nodeEndpoint: wss://kovan.infura.io/ws/v3/16a5aa3a08c24d56b1586cd06b4055d5
+    monitors:
+      syntheticPoolGeneral-FEUR:
+        task: synthetic.pools
+        arguments:
```

Comment:
can you make one task monitor all pools?

Reply:
> can you make one task monitor all pools?
Does it also include all tokens?

Reply:
few changes:

- make the token name can be an array, so user can pass `[FEUR, FJPY]`
- the SDK should include all token names
- add a alias so `all` translates to `[FEUR, FJPY, ...]` all tokens